verified-reviews/utils/client.ts

```diff
@@ -18,6 +18,7 @@ export interface PaginationOptions {
     | "rate_DESC"
     | "rate_ASC"
     | "helpfulrating_DESC";
+  useOrdenationCompatibility?: boolean;
```

Comment:
Set order as:

```
 order?:
    | "date_desc"
    | "date_ASC"
    | "rate_DESC"
    | "rate_ASC"
    | "helpfulrating_DESC"
    | string;
```

Add a comment explained that 
`"date_desc, date_ASC, rate_DESC, rate_ASC, helpfulrating_DESC"` are legacy compatible parameters that will be mapped to "date_desc, date_asc, rate_desc, rate_asc, helpfulrating_desc"`, but that we overall recommend the usage of a customized parameter.